cost-analyzer/templates/cost-analyzer-service-template.yaml

```diff
@@ -25,6 +25,15 @@ spec:
 {{- else }}
   type: ClusterIP
 {{- end }}
+{{- if (eq .Values.service.type "LoadBalancer") }}
+  {{- if .Values.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
```

Comment:
@itsmesuniljacob this looks like it was deprecated as of 1.24, do you think it still merits inclusion?

Reply:
```
 {{- if .Values.service.loadBalancerIP }}
  loadBalancerIP: {{ .Values.service.loadBalancerIP }}
```

I can remove this code

Reply:
Pushed the new change